### ISSUE TITLE
OMADA script does not work.

### ISSUE THREAD

**Issue by zee00100** [tteck/Proxmox#1051]
Hi.  Running the latest version of Proxmox (7.3-6).

Running the OMADA script does not work.  Upon running the script (regardless of choosing default or customized) it ends up......

   ____                      __     
  / __ \____ ___v5____ _____/ /___ _
 / / / / __  __ \/ __  / __  / __  /
/ /_/ / / / / / / /_/ / /_/ / /_/ / 
\____/_/ /_/ /_/\__,_/\__,_/\__,_/  
 
Using Default Settings
Using ubuntu Version: 20.04
Using Container Type: Unprivileged NO DEVICE PASSTHROUGH
Using Root Password: Automatic Login
Using Container ID: 110
Using Hostname: omada
Using Disk Size: 8GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Omada LXC using the above default settings
root@nuc:~# 

Basically dumps you back to the prompt.  

This does not happen with let's say the UniFi Script.

![image](https://user-images.githubusercontent.com/88451194/218340392-a2a55b4d-001f-4113-8678-87f8cfae3676.png)

**Comment by tteck** [comment]
Thanks for reporting.
Fixed

**Comment by zee00100** [comment]
Thanks!  Trying it now.

**Comment by zee00100** [comment]
Confirmed working!  It's generating the LXC as I type this!  Thanks for the super speedy fix!
